### PR TITLE
enable of "raft-engine": change the default value

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
Changed the default value of `enable` of "raft-engine".

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/7512
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by tabokie — APPROVED** [review]
I added a new line recently, could you translate that as well?
```
If you need to downgrade to a version earlier than v5.4.0, please disable Raft Engine and restart TiKV first.
```

**Review by TomShawn — APPROVED** [review]

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1343,14 +1343,14 @@ raftdb 相关配置项。
...
-
 ### `enable`
 
 + 决定是否使用 Raft Engine 来存储 Raft 日志。开启该配置项后，`raftdb` 的配置不再生效。
-+ 默认值：`"false"`
++ 默认值：`"true"`
+
+> **注意：**
+>
+> v5.4.0 版本的 Raft Engine 数据格式与之前版本不兼容。因此，把 TiDB 集群降级至 v5.4.0 以前的版本之前，需要先关闭 Raft Engine（即把 `enable` 配置项设置为 `false`）后重启 TiKV，否则会导致降级后集群无法开启。
```
Comment (en-jin19) [inline tikv-configuration-file.md:1353]: @TomShawn 该注意模块已更新，PTAL，谢谢！

Comment (en-jin19) [inline tikv-configuration-file.md:1353]:
```suggestion
> v5.4.0 版本的 Raft Engine 数据格式与之前版本不兼容。因此，在把 TiDB 集群降级至 v5.4.0 以前的版本**之前**，需要先关闭 Raft Engine（即把 `enable` 配置项设置为 `false`）后重启 TiKV，否则会导致降级后集群无法开启。
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1343,14 +1343,14 @@ raftdb 相关配置项。
...
-
 ### `enable`
 
 + 决定是否使用 Raft Engine 来存储 Raft 日志。开启该配置项后，`raftdb` 的配置不再生效。
-+ 默认值：`"false"`
++ 默认值：`"true"`
+
+> **注意：**
+>
+> v5.4.0 版本的 Raft Engine 数据格式与之前版本不兼容。因此，在把 TiDB 集群降级至 v5.4.0 以前的版本**之前**，需要先关闭 Raft Engine（即把 `enable` 配置项设置为 `false`）后重启 TiKV，否则会导致降级后集群无法开启。
```
Comment (en-jin19) [inline tikv-configuration-file.md:1353]:
```suggestion
> TiDB v5.4.0 版本的 Raft Engine 数据格式与之前版本不兼容。因此，如需将 TiDB 集群降级至 v5.4.0 以前的版本，你需要在降级**之前**先关闭 Raft Engine（即把 `enable` 配置项设置为 `false`），然后重启 TiKV，否则会导致集群降级后无法正常开启。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1343,14 +1343,14 @@ raftdb 相关配置项。
...
-
 ### `enable`
 
 + 决定是否使用 Raft Engine 来存储 Raft 日志。开启该配置项后，`raftdb` 的配置不再生效。
-+ 默认值：`"false"`
++ 默认值：`"true"`
+
+> **注意：**
+>
+> TiDB v5.4.0 版本的 Raft Engine 数据格式与之前版本不兼容。因此，当要将 TiDB 集群降级至 v5.4.0 以前的版本时，你需要在降级**之前**先关闭 Raft Engine（即把 `enable` 配置项设置为 `false`），并重启 TiKV 使配置生效，否则会导致集群降级后无法正常开启。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1353]: 这里放在 L1348 行前面会不会更好呢？

Comment (en-jin19) [inline tikv-configuration-file.md:1353]:
```suggestion
+ 
    > **注意：**
    >
    > TiDB v5.4.0 版本的 Raft Engine 数据格式与之前版本不兼容。因此，当要将 TiDB 集群降级至 v5.4.0 以前的版本时，你需要在降级**之前**先关闭 Raft Engine（即把 `enable` 配置项设置为 `false`），并重启 TiKV 使配置生效，否则会导致集群降级后无法正常开启。

+ 默认值：`"true"`
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**